osu.Game/Screens/Edit/ClipboardContent.cs

```diff
@@ -16,6 +16,8 @@ public class ClipboardContent
         [JsonConverter(typeof(TypedListConverter<HitObject>))]
         public IList<HitObject> HitObjects;
 
+        public const string CLIPBOARD_FORMAT = "osu/hitobjects";
```

Comment:
not a valid mime format, should probably be something like `application/x-osu-hitobjects`

Reply:
https://www.rfc-editor.org/rfc/rfc6838.html#section-3.4

> Subtype names with "x." as the first facet may be used for types
> intended exclusively for use in private, local environments.

https://www.rfc-editor.org/rfc/rfc6838.html#section-4.2.8

> Since this was published, the de facto practice has arisen for using
> this suffix convention for other well-known structuring syntaxes.  In
> particular, media types have been registered with suffixes such as
> "+der", "+fastinfoset", and **"+json"**.  This specification formalizes
> this practice and sets up a registry for structured type name
> suffixes.

So it should be `application/x.osu.hitobjects+json`.

Please note that web custom formats are prefixed with `web ` in the javascript API (an `image/png` and `web image/png` can be different things). I'm still thinking about how this should be handled.